diagnostic.js

```diff
@@ -30,25 +30,63 @@ mongoose.connect('mongodb://localhost/mongoose-crud');
 
 /// ADD YOUR CODE BELOW
 
-const create = (name, description, startYear, endYear) => {};
+const controller = require('lib/wiring/controller');
+const models = require('app/models');
+const Art = models.art;
```

Comment:
This code supposes that we're working within Antony's template, since it relies on Antony's `controller` code and his simplified file loader. We are _not_ using his template in this case.